src/tests/ftest/util/test_utils_container.py

```diff
@@ -356,8 +356,11 @@ def create(self, uuid=None, con_in=None, acl_file=None):
             }
 
             self._log_method("daos.container_create", kwargs)
-            uuid = self.daos.get_output("container_create", **kwargs)[0]
-
+            try:
+                uuid = self.daos.container_create(**kwargs)["results"]["uuid"]
+            except KeyError:
+                raise CommandFailure(
```

Comment:
(pylint-raise-missing-from) Consider explicitly re-raising using the 'from' keyword